### ISSUE TITLE
Request: Multiple Instances

### ISSUE THREAD

**Issue by deini** [adierkens/webpack-inject-plugin#19]
Unless I'm doing something wrong, if you do something like:

```js
{
  plugins: [
        new InjectPlugin(function() {
          return "console.log('Hello World');"
      }),
        new InjectPlugin(function() {
          return "console.log('Hello World 2');"
      })
  ]
}
```

It will just inject the first one. It would be cool if we can inject multiple. Somehow using the uniqueId for the module path.

**Comment by adierkens** [comment]
This is definitely something I can look into supporting, I think I know what's going on.

**Comment by adierkens** [comment]
Your PR was released in [1.1.0](https://github.com/adierkens/webpack-inject-plugin/releases/tag/1.1.0) 
Feel free to add yourself to the contributors list!
